test/runtests.jl

```diff
@@ -37,5 +36,5 @@ end
 # Perform Tests
 include("$(alpine_dir)/test/solver.jl")
 include("$(alpine_dir)/test/expression.jl")
-include("$(alpine_dir)/test/algorithm.jl")
-include("$(alpine_dir)/test/utility.jl")
+# include("$(alpine_dir)/test/algorithm.jl")
```

Comment:
Why are these commented ? This is the cause of the coverage drop

Reply:
@blegat Good point! I am just cleaning up tests individually. I just created the PR but still pushing updates to the PR. It should be uncommented in the newer commits.